server/@types/licenceApiClientTypes.ts

```diff
@@ -15,7 +15,11 @@ export type AdditionalConditionData = components['schemas']['AdditionalCondition
 export type ContactNumberRequest = components['schemas']['ContactNumberRequest']
 export type CreateLicenceRequest = components['schemas']['CreateLicenceRequest']
 export type LicenceSummary = components['schemas']['LicenceSummary']
-export type Licence = components['schemas']['Licence']
+export type Licence =
+  | components['schemas']['VariationLicence']
+  | components['schemas']['HardStopLicence']
+  | components['schemas']['CrdLicence']
```

Comment:
Had to manually delete a union from the generated licencesApiClientTypes and manually implement here to get round a bug where Licence and CrdLicence were treated as `any`.

This is hopefully due to a bug which will be fixed by [this](https://github.com/drwpow/openapi-typescript/pull/1578)